pyuvdata/uvdata/ms.py

```diff
@@ -1524,6 +1524,7 @@ def _read_ms_main(
         self.extra_keywords["DATA_COL"] = data_column
 
         time_arr = tb_main.getcol("TIME")
+        timescale = tb_main.getcolkeyword("TIME", "MEASINFO")["Ref"]
```

Comment:
Thanks. I updated the code to error by default if a non-supported timescale is detected, but that error can optionally be turned into a warning so that the file can still be read (in this case it just treats it as if it is UTC).

Reply:
FWIW, I think all of the non-natively supported timescales can be enabled with some "scaffolding" code. Not suggesting that we add such support from the get-go, but it may be worth adding a note in the warning that support for it can be requested on the GitHub issues page.